app/src/main/kotlin/com/wire/android/ui/authentication/login/LoginTopBar.kt

```diff
@@ -40,5 +35,5 @@ fun LoginTopBar() {
 @Preview(showBackground = false)
 @Composable
 fun LoginTopBarPreview() {
-    LoginTopBar()
+    LoginTopBar({})
```

Comment:
make preview fun `private` and remove useless imports plz

Reply:
🚀  Right, I keep forgetting about that